Sources/SwiftFrameCore/Extensions/DispatchQueue+Extensions.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+extension DispatchQueue {
+
+    func ky_asyncOrExit(_ block: @escaping () throws -> Void) {
+        async {
+            do {
+                try block()
+            } catch let error {
+                print(CommandLineFormatter.formatError("\(error.localizedDescription)"))
+                exit(1)
```

Comment:
Not sure if it's worth it but this seems to be the same behavior that we use for when exceptions on the main thread bubble up into the main method so maybe we could extract it, e.g. to an `ExceptionHandler` or something.

Reply:
Was thinking the same thing actually 👍